extension/src/cli/dvc/config.ts

```diff
@@ -21,6 +24,28 @@ export class DvcConfig extends DvcCli {
     return this.executeSafeProcess(cwd, Command.REMOTE, ...args)
```

Comment:
`executeSafeProcess` catches the error, stopping the external command code from [showing the error toast](https://github.com/iterative/vscode-dvc/blob/add-error-toasts-to-remote-commands/extension/src/commands/internal.ts#L83) and allowing the [Toast.showOutput](https://github.com/iterative/vscode-dvc/pull/5105/files#diff-d85871702cda3fd295552b55cb8c3131e5abbbe3c96f7b5beec7491905dd82daR156) used in `setup/index.ts` to get `undefined` (aka `Operation successful`). 

I've fixed that by adding functions for adding and modifying the remote that don't catch the error thrown.

Reply:
any other commands we need to move off safe process?

Reply:
I don't think so. Remote list commands are run periodically and remote remove commands are guaranteed to almost always fail since we run them twice in different scopes.